templates/Notifications/notifications.html.twig

```diff
@@ -0,0 +1,200 @@
+        </span>
+      </span>
+    </div>
+    <div id="follow-notif" class="mdc-chip chip" data-toggle="tab" href="#follow" role="tab">
+      <div class="mdc-chip__ripple"></div>
+      <span class="material-icons mdc-chip__icon mdc-chip__icon--leading">person</span>
+      <span role="gridcell">
+        <span role="button" tabindex="-1" class="mdc-chip__primary-action">
+          <span class="mdc-chip__text">{{ 'FollowersNotifications'|trans({}, 'catroweb') }}</span>
+        </span>
+      </span>
+    </div>
+    <div id="comment-notif" class="mdc-chip chip" data-toggle="tab" href="#comment" role="tab">
+      <div class="mdc-chip__ripple"></div>
+      <span class="material-icons mdc-chip__icon mdc-chip__icon--leading">comment</span>
+      <span role="gridcell">
+        <span role="button" tabindex="-2" class="mdc-chip__primary-action">
+          <span class="mdc-chip__text">{{ 'CommentNotifications'|trans({}, 'catroweb') }}</span>
+        </span>
+      </span>
+    </div>
+    <div id="reaction-notif" class="mdc-chip chip" data-toggle="tab" href="#reaction" role="tab">
+      <div class="mdc-chip__ripple"></div>
+      <span class="material-icons mdc-chip__icon mdc-chip__icon--leading">thumb_up</span>
+      <span role="gridcell">
+        <span role="button" tabindex="-3" class="mdc-chip__primary-action">
+          <span class="mdc-chip__text">{{ 'LikeNotifications'|trans({}, 'catroweb') }}</span>
+        </span>
+      </span>
+    </div>
+    <div id="remix-notif" class="mdc-chip chip" data-toggle="tab" href="#remix" role="tab">
+      <div class="mdc-chip__ripple"></div>
+      <span class="material-icons mdc-chip__icon mdc-chip__icon--leading">call_split</span>
+      <span role="gridcell">
+        <span role="button" tabindex="-4" class="mdc-chip__primary-action">
+          <span class="mdc-chip__text">{{ 'RemixNotifications'|trans({}, 'catroweb') }}</span>
+        </span>
+      </span>
+    </div>
+  </div>
+  <div class="tab-content mt-4 col-md-12">
+    <div id="all" class="tab-pane fade show active" role="tabpanel">
+      <div id="no-notifications"
+           class="text-center mb-5 {{ allNotifications is empty ? 'd-block' : 'd-none' }}">
+        {{ "notificationsReadMessage"|trans({}, "catroweb") }}
+      </div>
+      <div id="notifications" class="row mb-5">
+        {% for notification in allNotifications %}
+          {% if not (instance[notification.id] == null) and (instance[notification.id] == "follow" or  instance[notification.id] == "like"
+            or instance[notification.id] == "comment" or instance[notification.id] == "program"
+            or instance[notification.id] == "remix") %}
+            {% if instance[notification.id] == "follow" %}
+              <div
+                  onclick="notification.markAsRead('{{ notification.id }}', 'follow', '{{ notification.getSeen() }}', '{{ notification.follower.id }}')"
```

Comment:
without the implicit action in mark as read to redirect -all those most ifs here can be removed